js/animate.js

```diff
@@ -0,0 +1,56 @@
+const width =
+  window.innerWidth ||
+  document.documentElement.clientWidth ||
+  document.body.clientWidth;
+const height =
+  window.innerHeight ||
+  document.documentElement.clientHeight ||
+  document.body.clientHeight;
+const constant = width / (height * 2.3);
```

Comment:
NIT: constant can be confusing , could we look for more relatable word like if it is related to div, constantDiv or constantBox I am not sure, you can look into it